### ISSUE TITLE
The history object

### ISSUE THREAD

**Issue by jhaggle** [kurapan/EAST#56]
Printing out the keys of the history object returned by fit_generator using
print(history.history.keys())
gives the following:
dict_keys(['loss', 'pred_score_map_loss', 'pred_geo_map_loss', 'learning_rate', 'small_text_weight'])

How do you do to get pred_score_map_loss', 'pred_geo_map_loss', 'learning_rate' and 'small_text_weight in here? The default values are ['accuracy', 'loss', 'val_accuracy', 'val_loss'] and I don't see how you change this.

Furthermore, is there a way to get the validation info into the history object, as in the default?

**Comment by francotheengineer** [comment]
You're asking how do you get these loss values in the history object? Use `history.history['loss']` should give the value of that loss value for that fit session.
